### ISSUE TITLE
Load contract from github

### ISSUE THREAD

**Issue by jbentke** [slockit/piet#3]
Would be nice to have a feature to load contracts straight from github like https://github.com/TennaGraph/TennaGraph/blob/master/smart_contracts/contracts/VotingManager.sol

**Comment by hai-ko** [comment]
@jbentke that is already possible by using url parameters see https://blog.slock.it/analyzing-solidity-smart-contracts-with-piet-6db33a733e79 e.g. `https://piet.slock.it/?gitHubUser=gnosis&gitHubRepo=safe-contracts&subDir=contracts`

**Comment by jbentke** [comment]
a button would be nicer thow instead of messing directly with the url

**Comment by hai-ko** [comment]
Agreed, I'll add this feature. I did first the URL solution because this makes it possible to send someone a link that directly loads the contracts.

**Comment by KoenBal** [comment]
Sometimes working in chrome
https://piet.slock.it/?gitHubUser=gnosis&gitHubRepo=safe-contracts&subDir=contracts/common
https://piet.slock.it/?gitHubUser=celo-org&gitHubRepo=celo-monorepo&subDir=packages/protocol/contracts/stability

I often receive the notification:
> Error: Request failed with status code 403 while loading the GitHub repository. Are you trying to access a private repository?

While it is obviously not a private repo.

no pop-up or anything in Brave
https://piet.slock.it/?gitHubUser=celo-org&gitHubRepo=celo-monorepo&subDir=packages/protocol/contracts/stability

**Comment by KoenBal** [comment]
Oke, it seems related to the GitHub rate limit

> documentation_url: "https://developer.github.com/v3/#rate-limiting"
> message: "API rate limit exceeded for 92.223.89.138. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)"
